src/components/RegisterForm/RegisterForm.styled.js

```diff
@@ -1,4 +1,93 @@
+import React from 'react';
 import styled from 'styled-components';
+import backgroundImage from '@/images/bg-png.png'
+import FBLogo from '@/images/icons/FBLogo.svg'
+import GoogleLogo from '@/images/icons/GoogleLogo.svg'
+import ErrorIcon from '@/images/icons/IconStroke.svg'
+
+export const Background = styled('section')({
```

Comment:
Most styled components uses . notation in project, let's stick to one approach for it. 
```suggestion
export const Background = styled.section``
```